lib/components/sync-status/index.tsx

```diff
@@ -62,19 +62,17 @@ class SyncStatus extends Component<StateProps> {
           anchorEl={anchorEl}
           id={popoverId}
           onClose={this.handlePopoverClose}
-          unsyncedNoteIds={unsyncedNoteIds}
         />
       </div>
     );
   }
 }
 
 const mapStateToProps: S.MapState<StateProps> = ({
-  appState: state,
-  ui: { simperiumConnected },
+  ui: { simperiumConnected, unsyncedNoteIds },
 }) => ({
   simperiumConnected,
-  unsyncedNoteIds: state.unsyncedNoteIds,
+  unsyncedNoteIds,
```

Comment:
this can come out entirely, no? since the `<SyncStatusPopover />` not gets it itself?